__tests__/pages/webhooks/checkout-calculate-taxes.test.ts

```diff
@@ -9,14 +9,24 @@ import {
   mockRequest,
 } from "../../utils";
 
+import * as joseModule from "jose";
 import { NextApiRequest, NextApiResponse } from "next";
 import * as taxJarRequest from "taxjar/dist/util/request";
 import { Request } from "taxjar/dist/util/types";
 import * as calculateTaxes from "../../../pages/api/webhooks/checkout-calculate-taxes";
 
+jest.mock("next/dist/compiled/raw-body/index.js", () => ({
```

Comment:
It is not ideal solution, but I am currently waiting for feedback from saleor-app-sdk team, how we should handle this.